analysis/Sentiment_Analysis.py

```diff
@@ -1,18 +1,22 @@
-from analysis.analysis import Analysis
+from analysis.Analysis import Analysis
 import os
-import tensorflow.keras
-# TODO this can be removed once the word_index is saved and loaded in
-from tensorflow.keras.datasets import imdb
+os.environ['TF_CPP_MIN_LOG_LEVEL'] = '2' # suppresses tf info and warning logs
+from tensorflow.keras.models import load_model
+import tensorflow.python.keras.engine.base_layer_v1 # needed for compiling deliverable
+from tensorflow.keras.datasets.imdb import get_word_index
 from tensorflow.keras.preprocessing.text import text_to_word_sequence
-from tensorflow.keras.preprocessing import sequence
-
+from tensorflow.keras.preprocessing.sequence import pad_sequences
 
 class Sentiment_Analysis(Analysis):
+    
+    # model_path = os.path.join(os.path.dirname(__file__), 'sentiment_model.tf')
+    model_path = os.path.join(os.path.expanduser('~'), 'FRAT-models\\2_7_sentiment_model.tf')
```

Comment:
`os.path.expanduser('~')` is equivalent to your home path. `model_path` should be something similar to 'C:\Users\yourUser\FRAT-models\2_7_sentiment_model.tf'